device-types/DataDirect Networks/SFA400NVX-EDR.yaml

```diff
@@ -0,0 +1,95 @@
+---
+manufacturer: DataDirect Networks
+model: SFA400NV-EDR
+slug: sfa400nvx-edr
```

Comment:
All slugs should have the manufacturer appended. In this case, please append `datadirect-networks-`